komga-webui/src/locales/en.json

```diff
@@ -230,37 +230,37 @@
     "book_number": "Book number: {name}",
     "book_series": "Series: {name}",
     "button_import": "Import",
-    "comicrack_preambule_html": "You can import existing ComicRack Reading Lists in <code>.cbl</code> format.<br/>Komga will try to match the provided series and book number with series and books in your libraries.",
+    "comicrack_preambule_html": "When import existing ComicRack Reading Lists in <code>.cbl</code> format<br/>Komga tries to match the provided series and book number with series and books in your libraries.",
     "field_files_label": "ComicRack Reading Lists (.cbl)",
     "import_read_lists": "Import Read Lists",
     "imported_as": "Imported as {name}",
-    "results_preambule": "Result of the import is shown below. You can also check the unmatched books for each provided file.",
-    "size_limit": "Size should be less than {size} MB",
+    "results_preambule": "Result of the import is shown below. You can check the unmatched books for each provided file.",
+    "size_limit": "The size should be less than {size} MB",
     "tab_title": "Data Import"
   },
   "dialog": {
     "add_to_collection": {
       "button_create": "Create",
-      "card_collection_subtitle": "No series | 1 series | {count} series",
```

Comment:
Counting between 1-10.

Reply:
what do you mean by that ?

---

the figure notation is important for display size, "One" in other languages could be very long, and overflow.

Reply:
And "No" can't?

Reply:
Edit: Changing to numerals following https://material.io/design/communication/writing.html#principles

Reply:
as stated above, i don't think it should apply to 0/none. You wouldn't see Gmail saying "You have 0 messages", would you ?

Reply:
Revert please.